src/api/libcellml/variable.h

```diff
@@ -55,8 +55,20 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
     UnitsPtr getUnits() const;
 
     /**
+     * @brief Set the initial value for this variable using a string.
+     * Set the initial value for this variable using a string.
+     *
+     * @overload
+     * @sa getInitialValue
+     * @param initialValue the initial value to set.
```

Comment:
Indeed, @jonc125, the actual rendering will have to be checked. It's just that I have seen both "The" and "the", so this particular comment was more about consistency.